_includes/comparison.html

```diff
@@ -204,6 +221,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td>Kubernetes</td>
         <td>Kubernetes, VMs, ECS</td>
         <td>Kubernetes</td>
+        <td>Kubernetes</td>
```

Comment:
Kubernetes, VMs, Metal

Reply:
External workload support (non-Kubernetes)
https://docs.cilium.io/en/v1.12/gettingstarted/external-workloads/#external-workloads

Reply:
I can only find Installation in Kubernetes in the docs. The external workload support is covered by the "Mesh Expansion" feature